Permanent/Common/Base/SwiftUIViews/CustomAlertView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  CustomAlertView.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 30.08.2023.
+
+import SwiftUI
+
+/// Displays a simple two options alert with title and context
+///  - Parameters:
+///     - title: This is the tile of the alert
+///     - context: The body of the alert
+///     - confirmButtonText: The text of the confirm button
+///     - confirmed: Will be a Binding<Bool> which will be set when confirm button is pressed
+///  - Returns: Returns a view of a simple alert with two options as a View
+struct CustomAlertView: View {
+    @Environment(\.presentationMode) var presentationMode
+    @Binding var confirmed: Bool
+    var title: String
+    var context: String
+    var confirmButtonText: String
+    
+    var body: some View {
+        VStack(spacing: 32) {
+            VStack(spacing: 16) {
+                Text(title)
+                    .textStyle(RegularBoldTextStyle())
+                    .multilineTextAlignment(.center)
+                    .foregroundColor(.darkBlue)
+                Text(context)
+                    .textStyle(SmallXRegularTextStyle())
+                    .multilineTextAlignment(.center)
+                    .foregroundColor(.darkBlue)
+                    .frame(width: 216)
```

Comment:
Any reason you need to add the frame?

Reply:
The `.frame` sets the width of the context for this AlertView, without it, we should add padding with clear horizontal constraints.